app/views/metrics/index.html.erb

```diff
@@ -6,3 +6,130 @@
 <% content_for :javascripts do %>
   <%= javascript_include_tag "metrics" %>
 <% end %>
+
+<div class="ui pointing secondary tabular menu">
+  <a class="item active" data-tab="watchlist">Watchlist</a>
+  <a class="item" data-tab="risk metrics">Risk Metrics</a>
+</div>
+<div class="ui tab active" data-tab="watchlist">
+  Watchlist goes here
+</div>
+<div class="ui tab" data-tab="risk metrics">
+  <h6> 
+  	Select one or more of the following risk conditions you want to apply. 
+  </h6>
+  <h6>
+  	<i> 
+  		These conditions will make up your risk metrics, which will be used to identify students who are at risk in this course. 
+  	</i>
+  </h6>
+  <br>
+  <div class="ui form">
+  	<div class="metric">
+		<div class="inline field">
+		  <div class="ui checkbox">
+		    <input type="checkbox" tabindex="0" class="hidden">
+		  </div>
+	      Students who have used
+	      <span>
+		      <div class="ui selection dropdown">
+		        <input type="hidden" name="late days">
+		        <i class="dropdown icon"></i>
+		        <div class="default text">#</div>
+		        <div class="menu">
+		          <div class="item" data-value="1">1</div>
+		          <div class="item" data-value="2">2</div>
+		          <div class="item" data-value="3">3</div>
+		        </div>
+		  	  </div>
+	  	  </span>
+	  	  late days by
```

Comment:
Forgot to mention but we are using the term "Grace days" instead of "late days" to signify that these are penalty free extra days to use.